### PR TITLE
ci: Interop tests fixes and updates pending from #3331

### PR DESCRIPTION
**Pull request by jxs**

### CODE REVIEW

File: interop-tests/Cargo.toml [hunk 1]
```diff
@@ -7,14 +7,15 @@ publish = false
...
+rand = "0.8.5"
 redis = { version = "0.22.1", features = ["tokio-native-tls-comp", "tokio-comp"] }
+strum = { version = "0.24.1", features = ["derive"] }
 tokio = { version = "1.24.1", features = ["full"] }
 
 libp2p = { path = "../", default_features = false, features = ["websocket", "quic",  "mplex", "yamux", "tcp", "tokio", "ping", "noise", "tls", "dns", "rsa", "macros", "webrtc"] }
-rand = "0.8.5"
-strum = { version = "0.24.1", features = ["derive"] }
-either = "1.8.0"
+
```
Comment (jxs) [inline interop-tests/Cargo.toml]: >[thomaseizinger](https://github.com/thomaseizinger) [14 hours ago](https://github.com/libp2p/rust-libp2p/pull/3331#discussion_r1081904620)
Any reason for this newline?

was to separate libp2p from the other dependencies, meanwhile with `cargo add` that was lost, separated again with sorting them ptal.

Comment (thomaseizinger) [reply interop-tests/Cargo.toml]: > >[thomaseizinger](https://github.com/thomaseizinger) [14 hours ago](https://github.com/libp2p/rust-libp2p/pull/3331#discussion_r1081904620)
> Any reason for this newline?
> 
> was to separate libp2p from the other dependencies, meanwhile with `cargo add` that was lost, separated again with sorting them ptal. 

Do we need to separate libp2p? Isn't it just another dependency? :)

Comment (jxs) [reply interop-tests/Cargo.toml]: yeah, no strong opinion. This was something brought from `test-plans` repo where we have multiple `libp2p` versions.
Here it's only one so yeah you are probably right. Updated :)

**Comment by jxs** [comment]
btw, I am noticing the WebRTC test is flaky, sometimes it fails

**Review by thomaseizinger — APPROVED** [review]
Nice! Thank you for the follow-ups!

**Comment by thomaseizinger** [comment]
> btw, I am noticing the WebRTC test is flaky, sometimes it fails

Damn, do you have an error log? Which part is flaky? Our implementation? Do we just need longer timeouts? WebRTC handshakes can take a while ..

**Comment by thomaseizinger** [comment]
> > btw, I am noticing the WebRTC test is flaky, sometimes it fails
> 
> Damn, do you have an error log? Which part is flaky? Our implementation? Do we just need longer timeouts? WebRTC handshakes can take a while ..

Potentially relevant: https://github.com/libp2p/test-plans/issues/107

**Comment by jxs** [comment]
> > btw, I am noticing the WebRTC test is flaky, sometimes it fails
> 
> Damn, do you have an error log? Which part is flaky? Our implementation? Do we just need longer timeouts? WebRTC handshakes can take a while ..

yes sorry Thomas, Max also caught it https://github.com/libp2p/rust-libp2p/pull/3363#issuecomment-1398492669

**Pull request merged**